### ISSUE TITLE
[HOLD for payment 2025-01-17] [$100] Distance rates - Tax reclaimable can be equal Tax rates

### ISSUE THREAD

**Issue by vincdargento** [Expensify/App#53709]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v9.0.72-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N/A
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/cases/view/3073283
**Email or phone of affected tester (no customers):** N/A
**Issue reported by:** Applause Internal Team

## Action Performed:
Precondition: Distance rates and Taxes are enabled in workspace more features tab. Track tax is enabled in Distance rates settings.

1.  Go to WS > Distance rates
2. Click on Default rate ($0,67 / mile)
3. Set Tax Rate
4. Click on Tax reclaimable on
5. Set value > Default rate (e.g. 0.68) > Save
6. Set value = Default rate (e.g. 0.67) > Save

## Expected Result:
Step 5. An error appears 'The reclaimable portion must be less than the distance rate amount.'
Step 6. Value is not saved. Expect that 'The reclaimable portion must be less than the distance rate amount.' OR another error text. User can set 0.66 but not 0.67

## Actual Result:
Step 6. Value is saved.

## Workaround:
Unknown

## Platforms:
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/b863db73-69c2-4486-9ec8-7c659ebaf1a1

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021867265132758522754</li>
        <li>Upwork Job ID: 1867265132758522754</li>
        <li>Last Price Increase: 2024-12-12</li>
<li>Automatic offers: </li>
<ul>
<li>FitseTLT | Contributor | 105312886</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @muttmuure</details>

**Comment by FitseTLT** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Distance rates - Tax reclaimable can be equal Tax rates

### What is the root cause of that problem?
We are only checking that it is not greater here
https://github.com/Expensify/App/blob/48b80f65e80d304965cbeb3573d7d6678fb9bd3a/src/libs/PolicyDistanceRatesUtils.ts#L50-L51
### What changes do you think we should make in order to solve the problem?

We should change the condition to `>=` and update the copy accordingly
### What specific scenarios should we cover in automated tests to prevent reintroducing this issue in the future?


### What alternative solutions did you explore? (Optional)

**Comment by MonilBhavsar** [comment]
@muttmuure this an edge case and one liner change, could be tagged at $100

**Comment by mananjadhav** [comment]
@FitseTLT's [proposal](https://github.com/Expensify/App/issues/53709#issuecomment-2523678455) looks good to me.

🎀 👀 🎀 C+ reviewed.

**Comment by dangrous** [comment]
Yep proposal looks good! Given the testing steps and expectations, I don't think we want to allow it to be equal.

**Comment by mananjadhav** [comment]
@dangrous The PR is ready for your review.

**Comment by mananjadhav** [comment]
@dangrous This would be ready for payout soon. Earlier this deemed to be a single line change so we got the price to $100 can we bump the price now as we've also added tests for the same?

**Comment by mananjadhav** [comment]
@muttmuure quick bump on the previous comments.

**Comment by muttmuure** [comment]
Looks like we're just ready to pay out?

**Comment by muttmuure** [comment]
$100 - @mananjadhav C+
$100 - @FitseTLT C

**Comment by mananjadhav** [comment]
@muttmuure I think we're good to close here if the contributor has been paid.

**Comment by mananjadhav** [comment]
@muttmuure I think we're good to close here if the contributor has been paid out.

**Comment by muttmuure** [comment]
Paid

**Comment by garrettmknight** [comment]
$100 approved for @mananjadhav
